Packs/WolkenITSM/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Wolken ITSM",
+    "description": "Use The Wolken IT Service Management (ITSM) solution to modernize the way you manage and deliver services to your users.This is case management.Cortex XSOAR interfaces with Wolken ITSM to help streamline security-related service management and IT operations.\n\n",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Shubham Agarwal",
+    "url": "",
+    "email": "",
+    "created": "2022-02-11T04:23:34Z",
+    "categories": [],
```

Comment:
Please choose a category among the ones supported, see https://xsoar.pan.dev/docs/documentation/pack-docs#pack-keywords-tags-use-cases--categories.

Reply:
I have added categories field and save the changes. Please take a look once.